ios/magnet/AppDelegate.m

```diff
@@ -52,8 +54,17 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   UIViewController *rootViewController = [UIViewController new];
   rootViewController.view = rootView;
   self.window.rootViewController = rootViewController;
+  [NotificationsHelper register];
   [self.window makeKeyAndVisible];
   return YES;
 }
 
+- (void)applicationDidBecomeActive:(UIApplication *)application {
+  [NotificationsHelper clearNotifications];
+}
+
+- (void)applicationDidEnterBackground:(UIApplication *)application {
+  [NotificationsHelper clearNotifications];
```

Comment:
Good point, I though about it, but considered that is easier to clean on foreground and background, we will never get to an inconsistent state, but can dive into it.